packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -1504,3 +1500,18 @@ function isNullable(optionalValue: spec.OptionalValue | undefined): boolean {
         || (spec.isPrimitiveTypeReference(optionalValue.type)
             && optionalValue.type.primitive === spec.PrimitiveType.Any);
 }
+
+function paramJavadoc(name: string, optional?: boolean, summary?: string): string {
+    const parts = ['@param', name];
+    if (summary) { parts.push(endWithPeriod(summary)); }
+    if (!optional) { parts.push('This parameter is required.'); }
```

Comment:
Can we make this a bit more concise. Something like “(required)”

Reply:
I actually prefer "This parameter is required.". Yes, it's more verbose, but I think javadocs in libraries tend towards the verbose/descriptive side anyway, or at least do not try to be as short as possible. There is not much precedent for making javadocs more terse/concise that I can tell.

FWIW, others Java SDK team prefer ""This parameter is required." as well.

Reply:
Ok, cool.